gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/GradleCacheExplorer.java

```diff
@@ -57,17 +64,42 @@ public final Set<String> getCompletions(Set<String> repoUrls, DependencyGroup in
                 .flatMap(Optional::stream)
                 .collect(Collectors.toSet());
 
+        Set<String> resultsWithStarsIncluded = results.stream()
+                .flatMap(result -> includeStarsIfMultipleArtifacts(result, results))
```

Comment:
Honestly, I don't think we even need this logic for multiple artifacts. Even if there is just one artifact we should probably suggest `*`. At least for internal deps, I would prefer for people to start off with the `:*` as it means the star will be there when another package is added to that group.

Reply:
I've made it always suggest star, and updated the tests to match this new behaviour